Dockerfile

```diff
@@ -120,14 +120,14 @@ ARG BUILD_PROJECT=${RAY_JL_PROJECT}/build
 # Install custom Ray CLI which supports the Julia language.
 # https://docs.ray.io/en/releases-2.5.1/ray-contribute/development.html#building-ray-on-linux-macos-full
 ARG RAY_REPO=${HOME}/ray
-ARG RAY_COMMIT=448a83caf4
 ARG RAY_REPO_CACHE=/mnt/ray-cache
 ARG RAY_CACHE_CLEAR=false
+COPY --chown=${UID} build/ray-commit /tmp/ray-commit
```

Comment:
Docker build can't find the `/build/ray-commit` file
https://github.com/beacon-biosignals/Ray.jl/actions/runs/6342266769/job/17227631104?pr=164#step:4:248

Reply:
Whoops, used a hypen here